src/sql/workbench/contrib/dashboard/browser/core/dashboardTab.contribution.ts

```diff
@@ -157,3 +156,57 @@ ExtensionsRegistry.registerExtensionPoint<IDashboardTabContrib | IDashboardTabCo
 		}
 	}
 });
+
+const tabGroupSchema: IJSONSchema = {
+	type: 'object',
+	properties: {
+		id: {
+			type: 'string',
+			description: localize('azdata.extension.contributes.dashboard.tabGroup.id', "Unique identifier for this tab group.")
+		},
+		title: {
+			type: 'string',
+			description: localize('azdata.extension.contributes.dashboard.tabGroup.title', "Title of the tab group.")
+		}
+	}
+};
+
+const tabGroupContributionSchema: IJSONSchema = {
+	description: localize('azdata.extension.contributes.tabGroups', "Contributes a single or multiple tab groups for users to add to their dashboard."),
+	oneOf: [
+		tabGroupSchema,
+		{
+			type: 'array',
+			items: tabGroupSchema
+		}
+	]
+};
+
+ExtensionsRegistry.registerExtensionPoint<IDashboardTabContrib | IDashboardTabContrib[]>({ extensionPoint: 'dashboard.tabGroups', jsonSchema: tabGroupContributionSchema }).setHandler(extensions => {
+
+	function handleCommand(tabgroup: IDashboardTabGroupContrib, extension: IExtensionPointUser<any>) {
```

Comment:
Why is this named handleCommand? Doesn't seem to match what you're actually doing (handling tabGroups)

Reply:
:) updated